server/routes/api/trips/tripRoutes.js

```diff
@@ -24,20 +26,59 @@ router.post(
 
 router.get(
   '/cities',
+  authorizationCheck,
   allow('Requester', 'Manager'),
   tripController.getAllCities,
 );
 
 router.patch(
   '/:trip_id',
   [
-    allow('Requester', 'Manager'),
     authorizationCheck,
+    allow('Requester', 'Manager'),
     tripValidations.updateTripJoiValidation,
     tripVerification,
     tripValidations.checkOpenTripCityAndDATE,
   ],
   tripController.updateOpenTripDetails,
 );
 
+router.get('/', 
+    [
+      authorizationCheck,
+      allow('Requester', 'Manager'),
+      tripValidations.allTripsJOIValidation,
+      tripValidations.paginationByUserId,
+    ],
+    tripController.viewAllTrips,
+    );
+
+router.get('/assigned', 
```

Comment:
is assigned a correct identifier? Usually, the identifier should follow the collection name. eg. collectionName/:identifier. Here identifier is an identifier of a specific element/item/resource which is part of the collection.

Reply:
When I was implementing this task I thought `assigned` would be a good identifier, to indicate manager _assigned trips_.